### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by at-uytran**

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Api::V1::UsersController < ApplicationController
+  # GET /users
+  # skip_before_filter :verify_authenticity_token, only: %i[update create]
+  before_action :set_user, only: %i[show update destroy]
```
Comment (at-hoangho) [inline app/controllers/api/v1/users_controller.rb]: update?

File: app/controllers/api/v1/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: user_params[:email].downcase)
```
Comment (at-hoangho) [inline app/controllers/api/v1/sessions_controller.rb]: user_params[:email].downcase error if user_params[:email] = nil

File: db/migrate/20170822002734_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :user_name
+      t.string :first_name
+      t.string :last_name
+      t.string :password_digest
+      t.string :email
```
Comment (at-hoangho) [inline db/migrate/20170822002734_create_users.rb]: add index

File: app/controllers/api/v1/sessions_controller.rb (hunk 2 above)
Comment (at-toanngvan) [inline app/controllers/api/v1/sessions_controller.rb]: " ".downcase return " "

File: app/controllers/api/v1/users_controller.rb (hunk 1 above)
Comment (at-uytran) [inline app/controllers/api/v1/users_controller.rb]: edited

**Pull request closed without merging**